### DISCUSSION TITLE
Herramienta para creación de diagramas

### DISCUSSION THREAD

**Discussion by raulbernalcorral** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#76] [Ideas]
Hola explorers! Me estuve topando con restricciones de la versión gratuita en varias herramientas online para la creación de diagramas, por lo que les comparto una que en lo personal me ha servido mucho por su gran variedad de formas y tipos de diagramas (incluyendo UML que estamos requiriendo para esta primer práctica).

La herramienta se llama Dia, y la pueden descargar del siguiente link:

http://dia-installer.de/

![image](https://user-images.githubusercontent.com/15613043/155252435-d7b46047-c609-49a3-8b4d-4322911e4c84.png)

Saludos!

**Comment by manuel1488** [comment]
Gracias por compartir, creo que Microsfot Project que viene en algunas suite de Office te permite crear este tipo de diagramas tambien.
